CMakeLists.txt

```diff
@@ -49,6 +53,16 @@ option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
+option(USE_COTIRE "Use Cotire to speed up build (currently only for commands tests)" ON)
```

Comment:
@jacobkeeler could you please provide some information about fails? @jacobkeeler I have seen fails on travis. But I suppose that cotire requires bit amount of RAM  that cause error during linkage if memory in not enough.